concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -1,32 +1,54 @@
 import boto3
 import json
 import os
+import decimal
+from boto3.dynamodb import types
```

Comment:
The de/serializers are being accessed using `boto3.dynamodb.types`, so this import isn't being used at the moment.